cluster-sync/okd-4.3/hpp-master-mc.yaml

```diff
@@ -1,27 +0,0 @@
-apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
Can you open a PR against maybe the hostpath-provisioner-operator with the contents of these two files (hpp-master-mc.yaml and hpp-worker-mc.yaml). There have been quite a few times I have pointed people to these files with regards to Open Shift.

Reply:
Maybe add a contrib or something directory.

Reply:
Here: https://github.com/kubevirt/hostpath-provisioner-operator/pull/73

Reply:
thanks